apps/sharedtest/test/unit/l10n/lib/context/langnego_test.js

```diff
@@ -44,3 +44,20 @@ describe('Language negotiation with arguments', function() {
     assert.strictEqual(ctx.supportedLocales[1], 'en-US');
   });
 });
+
+describe('Language negotiation with arguments and unsupported', function() {
+  var ctx;
+
+  beforeEach(function(done) {
+    ctx = new Context();
+    ctx.once(done);
+    ctx.registerLocales('en-US', ['pl']);
+    ctx.requestLocales('pl-PT', 'pl');
```

Comment:
pl seems to be the one used in this directory, and in the fixtures sub-directory, there's only pl.properties and en-US.properties.

I also took a look, and de/fr aren't so convenient as there accept languages start off with the locale they are actually for.